code/Tasks/MigrateSiteTreeLinkingTask.php

```diff
@@ -35,11 +36,13 @@ public function run($request)
         Versioned::withVersionedMode(function () use (&$pages) {
             Versioned::set_stage(Versioned::DRAFT);
 
+            $sitetreeTbl = DataObject::singleton(SiteTree::class)->baseTable();
+
             /** @var SiteTree[] $linkedPages */
             $linkedPages = SiteTree::get()
                 ->innerJoin(
                     'SiteTree_LinkTracking',
-                    '"SiteTree_LinkTracking"."SiteTreeID" = "SiteTree"."ID"'
+                    "\"SiteTree_LinkTracking\".\"SiteTreeID\" = \"$sitetreeTbl\".\"ID\""
```

Comment:
I assume you’ve thought about this already, but do you want to use the variable for the first part of this name? The pivot table will always start with the base table name